src/components/inputsList/InputsList.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import Input from '../input/Input';
+
+const InputList = ({ inputs }) => (
+  <table>
+    <thead>
+      <tr>
+      {
+          inputs.length
+              ? renderHeaders()
+              : null
+      }
+      </tr>
+    </thead>
+    <tbody>
+    {
+        inputs.map(input => (
+            <Input key={input.id} {...input}  />
+        ))
+    }
+    </tbody>
+
+  </table>
+);
+
+export default InputList;
+
+function renderHeaders () {
+   let headers = [
```

Comment:
1) use const instead let
2) headers can and should be moved outside this functions cause no need to re-create this array on each renderHeaders call